node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/MetricsReporter.java

```diff
@@ -122,8 +122,7 @@ private void updateNodeMetrics(Node node, Map<HostName, List<ServiceInstance>> s
 
         metric.set("wantToRetire", node.status().wantToRetire() ? 1 : 0, context);
         metric.set("wantToDeprovision", node.status().wantToDeprovision() ? 1 : 0, context);
-        metric.set("hardwareFailure", node.status().hardwareFailureDescription().isPresent() ? 1 : 0, context);
-        metric.set("hardwareDivergence", node.status().hardwareDivergence().isPresent() ? 1 : 0, context);
+        metric.set("failReport", NodeFailer.reasonsToFailParentHost(node).isEmpty() ? 0 : 1, context);
```

Comment:
@hakonhall Need to update your nodes view dashboard

Reply:
👍